bokehjs/src/coffee/common/embed.coffee

```diff
@@ -8,6 +8,8 @@ HasProperties = require "./has_properties"
 {pull_session} = require "./client"
 {Promise} = require "es6-promise"
 
+documents = {}
+
```

Comment:
I think this global could in theory break things because it makes distinct embeds that would be isolated before now potentially share the same Document instance. Though real cases where that happens might be a little contrived, not sure.